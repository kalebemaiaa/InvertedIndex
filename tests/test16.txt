QueryAndS(QueryNot(QueryWord("deputado")), QueryOr(QueryWord("presidente"), QueryWord("ministro")))
11627.raw
3045.raw
3565.raw
805.raw
6287.raw
11079.raw
6013.raw
11591.raw
368.raw
11811.raw
2094.raw
11899.raw
11878.raw
2230.raw
8835.raw
5101.raw
1509.raw
2403.raw
907.raw
8881.raw
3685.raw
698.raw
1946.raw
6121.raw
11252.raw
1696.raw
1291.raw
4552.raw
973.raw
3923.raw
1874.raw
11599.raw
4479.raw
2084.raw
44.raw
3795.raw
11362.raw
3563.raw
4596.raw
1193.raw
423.raw
8887.raw
4145.raw
11820.raw
1195.raw
4729.raw
4474.raw
5187.raw
749.raw
433.raw
2341.raw
6098.raw
8751.raw
4017.raw
10975.raw
597.raw
6254.raw
1088.raw
1127.raw
11005.raw
915.raw
460.raw
5563.raw
4115.raw
1800.raw
4673.raw
14.raw
5532.raw
4081.raw
2926.raw
4427.raw
4581.raw
10895.raw
3529.raw
652.raw
4494.raw
3884.raw
1988.raw
4603.raw
975.raw
2079.raw
3417.raw
1654.raw
5818.raw
5837.raw
4923.raw
2849.raw
2044.raw
32.raw
11208.raw
4950.raw
11111.raw
11849.raw
3097.raw
2919.raw
3426.raw
3280.raw
135.raw
2360.raw
4340.raw
11740.raw
2156.raw
6237.raw
517.raw
2205.raw
5287.raw
6231.raw
24.raw
2795.raw
11640.raw
5487.raw
2586.raw
3656.raw
4002.raw
11458.raw
2204.raw
1630.raw
3296.raw
11579.raw
1821.raw
3170.raw
1676.raw
11720.raw
11389.raw
11043.raw
1992.raw
900.raw
2687.raw
6257.raw
4406.raw
2938.raw
1679.raw
6265.raw
750.raw
11861.raw
5883.raw
2266.raw
8828.raw
4123.raw
5190.raw
2157.raw
1243.raw
5504.raw
11859.raw
3530.raw
3654.raw
3299.raw
3534.raw
6367.raw
2126.raw
3524.raw
10839.raw
11336.raw
90.raw
2754.raw
3584.raw
3227.raw
1623.raw
1432.raw
4166.raw
2454.raw
11029.raw
1925.raw
3506.raw
4271.raw
6391.raw
61.raw
4696.raw
4831.raw
3326.raw
1959.raw
3818.raw
1058.raw
6131.raw
11468.raw
3032.raw
11668.raw
6392.raw
18.raw
5237.raw
2550.raw
551.raw
2828.raw
2551.raw
11003.raw
2808.raw
11048.raw
4066.raw
4003.raw
1279.raw
2525.raw
3941.raw
1980.raw
11212.raw
2702.raw
3575.raw
3176.raw
5513.raw
3916.raw
1913.raw
