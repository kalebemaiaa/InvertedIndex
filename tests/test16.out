14
18
24
44
135
368
460
597
698
749
907
915
1058
1088
1127
1193
1195
1509
1623
1676
1679
1874
1959
1980
1988
1992
2044
2084
2126
2156
2157
2204
2205
2230
2266
2341
2360
2454
2586
2687
2702
2754
2795
2808
2849
2919
2938
3097
3170
3227
3280
3296
3524
3529
3534
3563
3654
3685
3795
3916
3941
4066
4081
4115
4340
4406
4427
4479
4581
4596
4673
4696
4729
4950
5287
5487
5504
5532
5563
6231
6237
6265
6287
6367
8751
8828
8887
10839
11003
11005
11029
11043
11048
11079
11111
11212
11336
11389
11458
11640
11740
11811
11861
